### PR TITLE
Add instructions for running version-skewed tests

### LINKED ISSUE #27224: Add instructions for manually recreating upgrade tests
Per https://github.com/kubernetes/kubernetes/issues/26662#issuecomment-224678472.

### PR DESCRIPTION
**Pull request by ikehz**
Fixes #27224.

cc @thockin per your request.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/27555)

### CODE REVIEW

File: docs/devel/e2e-tests.md [hunk 1]
```diff
@@ -406,6 +406,70 @@ To control the tests that are run:
...
+cd ./kubernetes_old
+go run ./hack/e2e.go -v --up
+
+# Upgrade the cluster to the new version
+#
+# If using GKE, add --upgrade-target=${NEW_VERSION}
+#
+# You can target Feature:MasterUpgrade or Feature:ClusterUpgrade
+cd ../kubernetes
+go run ./hack/e2e.go -v --test --check_version_skew=false --test_args="ginkgo.focus=\[Feature:MasterUpgrade\]"
```
Comment (jingxu97) [inline docs/devel/e2e-tests.md]: Is there -- before ginkgo.focus?

**Comment by ihmccreery** [comment]
Yes, thanks.  Fixed; PTAL.

**Comment by jingxu97** [comment]
Some failures:
FAIL: changes needed but not made due to --verify
/go/src/k8s.io/kubernetes/docs/ is out of date. Please run hack/update-munge-docs.sh
FAILED   ./hack/../hack/verify-munge-docs.sh	2s
Verifying ./hack/../hack/verify-swagger-spec.sh

What is the issue?

**Comment by jingxu97** [comment]
ok to test

**Comment by jingxu97** [comment]
fixed by #29859

**Pull request closed without merging**